blocks/library/gallery/index.js

```diff
@@ -121,16 +124,24 @@ registerBlockType( 'core/gallery', {
 			controls,
 			focus && images.length > 1 && (
 				<InspectorControls key="inspector">
+					<p className="editor-block-inspector__description">Image galleries are a great way to share groups of pictures on your site.</p>
+					<hr />
```

Comment:
I figured we'd keep it manual for now, per the idea that we might have blocks with no inspector options, or even use the description elsewhere. 

By the way, I've used this pattern for the design:

![screen shot 2017-06-28 at 13 02 11](https://user-images.githubusercontent.com/1204802/27634064-0e098cc4-5c02-11e7-9102-8885730226b2.png)

The italics will come in when the Read More block is merged.